app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -11,53 +10,56 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       this.element = $element[0]
       this.somethingIsWrongForm = this.element.querySelector('#something-is-wrong')
       this.surveyForm = this.element.querySelector('#page-is-not-useful')
-      this.$prompt = $element.find('.js-prompt')
-      this.$fields = $element.find('.gem-c-feedback__form-field')
-      this.$forms = $element.find('.js-feedback-form')
-      this.$toggleForms = $element.find('.js-toggle-form')
-      this.$closeForms = $element.find('.js-close-form')
+      this.prompt = this.element.querySelector('.js-prompt')
+      this.forms = this.element.querySelectorAll('.js-feedback-form')
+      this.toggleForms = this.element.querySelectorAll('.js-toggle-form')
+      this.closeForms = this.element.querySelectorAll('.js-close-form')
       this.activeForm = false
-      this.$activeForm = false
-      this.$pageIsUsefulButton = $element.find('.js-page-is-useful')
-      this.$pageIsNotUsefulButton = $element.find('.js-page-is-not-useful')
-      this.$somethingIsWrongButton = $element.find('.js-something-is-wrong')
-      this.$promptQuestions = $element.find('.js-prompt-questions')
-      this.$promptSuccessMessage = $element.find('.js-prompt-success')
-      this.$somethingIsWrongForm = $(this.somethingIsWrongForm)
-      this.$surveyForm = $(this.surveyForm)
-      this.$surveyWrapper = $element.find('#survey-wrapper')
+      this.pageIsUsefulButton = this.element.querySelector('.js-page-is-useful')
+      this.pageIsNotUsefulButton = this.element.querySelector('.js-page-is-not-useful')
+      this.somethingIsWrongButton = this.element.querySelector('.js-something-is-wrong')
+      this.promptQuestions = this.element.querySelectorAll('.js-prompt-questions')
+      this.promptSuccessMessage = this.element.querySelector('.js-prompt-success')
+      this.surveyWrapper = this.element.querySelector('#survey-wrapper')
 
       var that = this
```

Comment:
I wonder if the `that` variable could be better named to indicate that it's the `this` for the whole module?

Reply:
I think this is a convention we use elsewhere, but do you have a suggestion?

Reply:
In this instance, would `thisModule` describe what `that` is? I've seen `var that = this` used, but there doesn't seem to be consistency about what `that` is - sometimes it's used for the whole module and sometimes a function inside the module. 

Not a blocker, just a thought.

Reply:
Yeah, I like that. Changed.